packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -802,6 +802,39 @@ class ListTile extends StatelessWidget {
       ),
     );
   }
+
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
+    super.debugFillProperties(properties);
+    properties.add(DiagnosticsProperty<Widget>('leading', leading, defaultValue: null));
+    properties.add(DiagnosticsProperty<Widget>('title', title, defaultValue: null));
+    properties.add(DiagnosticsProperty<Widget>('subtitle', subtitle, defaultValue: null));
+    properties.add(DiagnosticsProperty<Widget>('trailing', trailing, defaultValue: null));
+    properties.add(DiagnosticsProperty<bool>('isThreeLine', isThreeLine, defaultValue: false));
+    properties.add(DiagnosticsProperty<bool>('dense', dense, defaultValue: null));
+    properties.add(DiagnosticsProperty<VisualDensity>('visualDensity', visualDensity, defaultValue: null));
+    properties.add(DiagnosticsProperty<ShapeBorder>('shape', shape, defaultValue: null));
+    properties.add(DiagnosticsProperty<ListTileStyle>('style', style, defaultValue: null));
+    properties.add(DiagnosticsProperty<Color>('selectedColor', selectedColor, defaultValue: null));
+    properties.add(DiagnosticsProperty<Color>('iconColor', iconColor, defaultValue: null));
+    properties.add(DiagnosticsProperty<Color>('textColor', textColor, defaultValue: null));
+    properties.add(DiagnosticsProperty<EdgeInsetsGeometry>('contentPadding', contentPadding, defaultValue: null));
+    properties.add(DiagnosticsProperty<bool>('enabled', enabled, defaultValue: true));
+    properties.add(DiagnosticsProperty<Function>('onTap', onTap, defaultValue: null));
+    properties.add(DiagnosticsProperty<Function>('onLongPress', onLongPress, defaultValue: null));
+    properties.add(DiagnosticsProperty<MouseCursor>('mouseCursor', mouseCursor, defaultValue: null));
+    properties.add(DiagnosticsProperty<bool>('selected', selected, defaultValue: false));
+    properties.add(DiagnosticsProperty<Color>('focusColor', focusColor, defaultValue: null));
+    properties.add(DiagnosticsProperty<Color>('hoverColor', hoverColor, defaultValue: null));
+    properties.add(DiagnosticsProperty<FocusNode>('focusNode', focusNode, defaultValue: null));
+    properties.add(DiagnosticsProperty<bool>('autofocus', autofocus, defaultValue: false));
+    properties.add(DiagnosticsProperty<Color>('tileColor', tileColor, defaultValue: null));
+    properties.add(DiagnosticsProperty<Color>('selectedTileColor', selectedTileColor, defaultValue: null));
+    properties.add(DiagnosticsProperty<bool>('enableFeedback', enableFeedback, defaultValue: null));
+    properties.add(DiagnosticsProperty<double>('horizontalTitleGap', horizontalTitleGap, defaultValue: null));
```

Comment:
These should be `DoubleProperty` instead of `DiagnosticsProperty<double>`. This is likely the source of your problem.

Reply:
Thank you so much! I really appreciate your help!